app/pages/ServiceDiscoveryResults/SearchResults/SearchResults.scss

```diff
@@ -105,6 +105,5 @@
   .sideLinkText {
     padding: 0 8px;
     white-space: nowrap;
-    overflow: scroll;
```

Comment:
You might want to set this to [`overflow: auto`](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow) instead. `overflow: auto` will add scrollbars only if the content is too wide to fit in the containing element. If someone does have a very long phone number, we probably want to allow people to scroll to see all of it, even if it doesn't look as great. Setting `overflow: auto` will at least make it look normal when the content does fit, though.

Reply:
got it. that makes sense